### PR TITLE
changing urls for simple-web-worker repo

### PR DESCRIPTION
**Pull request by Rumyra**
#### Summary

We've moved the simple-web-worker example into dom-examples, so this updates the links 

#### Motivation

Tidying up repos - see https://github.com/mdn/dom-examples/pull/101

### CODE REVIEW

File: files/en-us/web/api/dedicatedworkerglobalscope/postmessage/index.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ onmessage = function(e) {
 
 In the main script, `onmessage` would have to be called on a `Worker object`, whereas inside the worker script you just need `onmessage` because the worker is effectively the global scope ({{domxref("DedicatedWorkerGlobalScope")}}).
 
-For a full example, see our [Basic dedicated worker example](https://github.com/mdn/simple-web-worker) ([run dedicated worker](https://mdn.github.io/simple-web-worker/)).
+For a full example, see our [Basic dedicated worker example]((https://github.com/mdn/dom-examples/tree/master/web-workers/simple-web-worker)) ([run dedicated worker]((https://mdn.github.io/dom-examples/web-workers/simple-web-worker/))).
```
Comment (Rumyra) [inline files/en-us/web/api/dedicatedworkerglobalscope/postmessage/index.md:64]:
```suggestion
For a full example, see our [Basic dedicated worker example](https://github.com/mdn/dom-examples/tree/master/web-workers/simple-web-worker) ([run dedicated worker](https://mdn.github.io/dom-examples/web-workers/simple-web-worker/)).
```

File: files/en-us/web/api/web_workers_api/index.md [hunk 2]
```diff
@@ -75,7 +75,7 @@ Workers can also spawn other workers, so these APIs are also available: {{domxre
 
 We have created a couple of simple demos to show basic usage:
 
-- [Basic dedicated worker example](https://github.com/mdn/simple-web-worker) ([run dedicated worker](https://mdn.github.io/simple-web-worker/)).
+- [Basic dedicated worker example]((https://github.com/mdn/dom-examples/tree/master/web-workers/simple-web-worker)) ([run dedicated worker]((https://mdn.github.io/dom-examples/web-workers/simple-web-worker/))).
```
Comment (Rumyra) [inline files/en-us/web/api/web_workers_api/index.md:78]:
```suggestion
- [Basic dedicated worker example](https://github.com/mdn/dom-examples/tree/master/web-workers/simple-web-worker) ([run dedicated worker](https://mdn.github.io/dom-examples/web-workers/simple-web-worker/)).
```

File: files/en-us/web/api/worker/index.md [hunk 3]
```diff
@@ -70,7 +70,7 @@ first.onchange = function() {
 }
 ```
 
-For a full example, see our [Basic dedicated worker example](https://github.com/mdn/simple-web-worker) ([run dedicated worker](https://mdn.github.io/simple-web-worker/)).
+For a full example, see our [Basic dedicated worker example]((https://github.com/mdn/dom-examples/tree/master/web-workers/simple-web-worker)) ([run dedicated worker]((https://mdn.github.io/dom-examples/web-workers/simple-web-worker/))).
```
Comment (Rumyra) [inline files/en-us/web/api/worker/index.md:73]:
```suggestion
For a full example, see our [Basic dedicated worker example](https://github.com/mdn/dom-examples/tree/master/web-workers/simple-web-worker) ([run dedicated worker](https://mdn.github.io/dom-examples/web-workers/simple-web-worker/)).
```

File: files/en-us/web/api/worker/postmessage/index.md [hunk 4]
```diff
@@ -62,7 +62,7 @@ second.onchange = function() {
 }
 ```
 
-For a full example, see our [simple worker example](https://github.com/mdn/simple-web-worker) ([run example](https://mdn.github.io/simple-web-worker/)).
+For a full example, see our [simple worker example]((https://github.com/mdn/dom-examples/tree/master/web-workers/simple-web-worker)) ([run example]((https://mdn.github.io/dom-examples/web-workers/simple-web-worker/))).
```
Comment (Rumyra) [inline files/en-us/web/api/worker/postmessage/index.md:65]:
```suggestion
For a full example, see our [simple worker example](https://github.com/mdn/dom-examples/tree/master/web-workers/simple-web-worker) ([run example](https://mdn.github.io/dom-examples/web-workers/simple-web-worker/)).
```

File: files/en-us/web/api/worker/worker/index.md [hunk 5]
```diff
@@ -60,7 +60,7 @@ first.onchange = function() {
 }
 ```
 
-For a full example, see our [Basic dedicated worker example](https://github.com/mdn/simple-web-worker) ([run dedicated worker](https://mdn.github.io/simple-web-worker/)).
+For a full example, see our [Basic dedicated worker example]((https://github.com/mdn/dom-examples/tree/master/web-workers/simple-web-worker)) ([run dedicated worker]((https://mdn.github.io/dom-examples/web-workers/simple-web-worker/))).
```
Comment (Rumyra) [inline files/en-us/web/api/worker/worker/index.md:63]:
```suggestion
For a full example, see our [Basic dedicated worker example](https://github.com/mdn/dom-examples/tree/master/web-workers/simple-web-worker) ([run dedicated worker](https://mdn.github.io/dom-examples/web-workers/simple-web-worker/)).
```

**Review by teoli2003 — APPROVED** [review]
This sounds good. Verified the example was actually moved. Thank you!

**Pull request merged**